src/core/index.ts

```diff
@@ -326,13 +354,31 @@ export class Context {
             });
         }
     }
+
+    hasFeature(feature: SDKFeature) {
+        if (!this.features) {
```

Comment:
Rather than delegate this here, perhaps during construction we could provide a default like `config.features ?? [PUSH]`?

Reply:
Yeah, was from spiking - will refine 👍